docs/deploy/terraform.md

```diff
@@ -0,0 +1,77 @@
+---
+seotitle: Integrate Encore with existing infrastructure
+seodesc: The Encore terraform provider lets you integrate your encore deployment with existing infrastructure
+title: Terraform Provider
+subtitle: Integrate Encore with existing infrastructure
+infobox: {
+  title: "Terraform Provider",
+  import: "https://registry.terraform.io/providers/encoredev/encore",
+}
+---
+Encore simplifies the deployment and management of cloud applications. When working with complex systems, you often 
+need to integrate Encore-provisioned resources into your broader infrastructure landscape. Terraform data sources 
```

Comment:
Maybe would be useful to clarify that this is something that Encore provides. Something like:

```suggestion
need to integrate Encore-provisioned resources into your broader infrastructure landscape. For this purpose, Encore maintains a Terraform Provider with data sources that
```